spotpass/3ds-task-finder.js

```diff
@@ -0,0 +1,59 @@
+const https = require('node:https');
+const axios = require('axios');
+const fs = require('fs-extra');
+const database = require('./database');
+const apps = require('./ctr-boss-apps.json');
+
+const NPFL_URL_BASE = 'https://npfl.c.app.nintendowifi.net/p01/filelist';
+const TASK_SEARCH = 'FGONLYT'; // * For this example, I'm using the common FGONLYT task name, but there are other common task names that can be tested.
```

Comment:
I'm curious what other common task names you've noticed? `FGONLYT` is the only one that comes to mind for me, which I assume stands for "***F***ore***g***round ***only*** ***t***ask"

Reply:
`news` is another one I noticed that's extremely common.

Reply:
You can probably find a few more by just checking the frequency at each task name shows up

Reply:
Update: Just wrote a quick script to check this. These are the tasks which show up more than 10 times (checked both Wii U and 3DS tasks)

- `FGONLYT`
- `news`
- `data`
- `TASK00`
- `0000001`

Reply:
Nice, thanks. Should be implemented now.